io.trezor.suite.yml

```diff
@@ -0,0 +1,72 @@
+app-id: io.trezor.suite
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+command: trezor-suite
+separate-locales: false
+rename-icon: trezor-suite
+finish-args:
+# Essential to get the autostart working.
+  - --filesystem=~/.config/autostart
```

Comment:
it should use the dynamic launcher portal instead of poking filesystem holes. Is autostart really needed?

Reply:
Would it work? In the code, it is implemented like this: https://github.com/trezor/trezor-suite/blob/71da008f500b198b916ae6731cb4b2d545720559/packages/suite-desktop-core/src/modules/auto-start.ts

The autostart is not that much needed yet, but it will become more important for the app as it facilitates the connection to other apps wanting to use Trezor devices.

Reply:
No that doesn't use the portal. If you use the portal https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.DynamicLauncher.html#org-freedesktop-portal-dynamiclauncher-install you can install desktop files without any filesystem access.

Reply:
Do I understand correctly that this is something that would need to be implemented upstream? If so, I'm worried it won't be possible due to the developers' constraints. I can remove it, but the app would lose one of its important functionalities. 
If it is something that can be added to the Flatpak only, I'm willing to explore this. I'm a bit new to this, so sorry for some of my confusion. :)

Reply:
> Do I understand correctly that this is something that would need to be implemented upstream? 

Yea

You can keep the permission if autostarting is integral to the app's main functionality. Like a reminder/alarm app must autostart etc.

Reply:
I will delete it for now as it is currently still an experimental feature. However, in the future, the "silent" autostart should be used as a way for people to have their Trezor devices recognized while using web-based wallets or other desktop apps. I assume it is possible to reopen the discussion when the issue becomes more relevant, right?

Reply:
You can add it back later but I'd prefer that dynamic launcher portal support is added by then.